omnibus/Gemfile.lock

```diff
@@ -1,6 +1,6 @@
 GIT
   remote: https://github.com/chef/omnibus-software.git
-  revision: 8a0e391d4232ab3fc5fb7a03b15932335a0214c4
```

Comment:
@johnmccrae Should we also create and see adhoc build passes before merging, due to updates in omnibus/Gemfile.lock?

Reply:
Great point. I'll roll one after I update with Marc's comments